search.py

```diff
@@ -70,16 +70,18 @@ def get_creator(tree):
     return creators_list
 
 
-def find_expressive_term(tree, expressive_term):
+def find_expressive_term(root, expressive_term):
     """return a list of elements that has expressive term that is of expressive_term"""
-    all_et_list = get_elements(tree, 'dir')
-    element_et_list = [element for element in all_et_list if element.text == expressive_term]
+    music = root.find("{http://www.music-encoding.org/ns/mei}music")
+    et_test = music.iter("{http://www.music-encoding.org/ns/mei}dir")
+    element_et_list = [element for element in et_test if element.text == expressive_term]
     return element_et_list
```

Comment:
This can be returned directly without create a new variable:

```javascript
return [element for element in et_test if element.text == expressive_term]
```